src/renderer/components/monaco-editor/monaco-editor.tsx

```diff
@@ -186,7 +186,14 @@ export class MonacoEditor extends React.Component<MonacoEditorProps> {
     if (!this.containerElem || this.editor || this.unmounting) {
       return;
     }
-    const { language, theme, readOnly, value: defaultValue } = this.props;
+    const {
```

Comment:
How this file changes related to PR's topic?

Reply:
Specifically the addition of the `fireImmediately` on line 220 below is needed. Without it the contents of the editor wasn't updating correctly.

Reply:
Okay, so then leave only `fireImmediately` here :)

Reply:
Okay